### ISSUE TITLE
lvm2-lvmdbusd.service fails to start

### ISSUE THREAD

**Issue by pohly** [clearlinux/distribution#348]
LVM is partly (?)  broken. I have installed `storage-utils` which contains `lvm2-lvmdbusd.service` on Clear Linux 27050. `ceph-volume lvm create` hangs for a long time (> 10 minutes) before it eventually proceeds (probably because of some internal timeout). I suspect the reason is this:

```
Jan 07 16:23:20 host-1 systemd[1]: Starting LVM2 D-Bus service...
Jan 07 16:23:20 host-1 lvmdbusd[22290]: Traceback (most recent call last):
Jan 07 16:23:20 host-1 lvmdbusd[22290]:   File "/usr/bin/lvmdbusd", line 13, in <module>
Jan 07 16:23:20 host-1 lvmdbusd[22290]:     from lvmdbusd import main
Jan 07 16:23:20 host-1 lvmdbusd[22290]: ModuleNotFoundError: No module named 'lvmdbusd'
Jan 07 16:23:20 host-1 systemd[1]: lvm2-lvmdbusd.service: Main process exited, code=exited, status=1/FAILURE
Jan 07 16:23:20 host-1 systemd[1]: lvm2-lvmdbusd.service: Failed with result 'exit-code'.
Jan 07 16:23:20 host-1 systemd[1]: Failed to start LVM2 D-Bus service.
```

**Comment by bryteise** [comment]
@pohly So it looks like ceph really wants the LVM python bindings. Hopefully that will make this issue go away for you.

**Comment by pohly** [comment]
@bryteise Is `lvm2-lvmdbusd.service` from Ceph? I've not checked. I thought it was a core LVM2 thing, based on the name.

**Comment by bryteise** [comment]
@pohly sorry I wasn't clear, it looks like the ceph-volume lvm create command is starts an LVM2 service (lvm2-lvmdbusd) that requires the LVM2 python bindings. We really don't want python in storage-utils though. I am going to change the storage-cluster bundle to add the LVM2 python bindings though I'm still not certain exactly how to handle the storage-utils bundle and LVM as figuring out what does and what does not need the python bindings is a bit tricky.

**Comment by pohly** [comment]
William Douglas <notifications@github.com> writes:
> I am going to change the storage-cluster bundle to add the
> LVM2 python bindings

That works for me. I had another look and in fact, I do install
storage-cluster to get Ceph. storage-utils was installed as a workaround
for some (now fixed) Clear Linux bug (#127) and can be removed.

> though I'm still not certain exactly how to
> handle the storage-utils bundle and LVM as figuring out what does and
> what does not need the python bindings is a bit tricky.

I don't know either. Are you removing lvm2-lvmdbusd.service from
storage-utils? There's no point in having a service that cannot run.

**Comment by bwarden** [comment]
`lvm2-lvmdbusd.service` is no longer packaged.
